compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraToMapLibre.kt

```diff
@@ -21,6 +22,18 @@ internal fun MapViewCamera.toCameraPosition(): CameraPosition {
           .tilt(this.state.pitch)
           .bearing(this.state.direction)
     }
+    is CameraState.BoundingBox -> {
+      return map.getCameraForLatLngBounds(
```

Comment:
Easy to miss, but noting here that we explicitly return early. The `CameraPosition` builder does not have any way to construct a bounding box position. You have to directly ask the map for this, so this is a different flow. The padding logic at the end is also skipped because this method is supposed to take care of the padding.

Reply:
Makes sense. We should throw this in there as a comment 👍